addons/uom/views/uom_uom_views.xml

```diff
@@ -62,6 +62,9 @@
                 <field name="name"/>
                 <separator/>
                 <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
+                <group string="Group By">
+                  <filter string="Category" name="group_by_category" domain="[]" context="{'group_by': 'category_id'}"/>
```

Comment:
Why the `domain="[]"` ? It has no impact.

Reply:
Because We are using It everywhere, I don't Know why.